code-challenges.test.js

```diff
@@ -15,40 +15,104 @@
 
 // a) Create a test with an expect statement using the variable provided. HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
+
+// Create Jest test using syntax example from syllubus with describe, it, and expect statements.
+describe("shuffled", () => {
+
 var colors1 = ["purple", "blue", "green", "yellow", "pink"]
-// Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
+  // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
 var colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
-// Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+  // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+
+  it("takes an array of strings, removes index 0, and shuffles the rest", () => {
+
+  // set expect statement to equal the array, with index 0 deleted, and the rest shuffled.
+    expect(["blue", "green", "yellow", "pink", "new"]).toEqual(expect.arrayContaining(shuffled(colors1)))
+    expect(["indigo", "periwinkle", "ochre", "aquamarine", "saffron"]).toEqual(expect.arrayContaining(shuffled(colors2)))
 
+  })
+})
```

Comment:
Keep an eye on the inputs here. Make sure the test input is what is passed to the function.